android/src/toga_android/widgets/selection.py

```diff
@@ -89,6 +89,3 @@ def rehint(self):
         )
         self.interface.intrinsic.width = at_least(self.native.getMeasuredWidth())
         self.interface.intrinsic.height = self.native.getMeasuredHeight()
-
-    def set_alignment(self, value):
-        self.native.setGravity(Gravity.CENTER_VERTICAL | align(value))
```

Comment:
This had no effect because the gravity we actually need to change is that of the TextView within the `R__layout.simple_spinner_item`. This can be fixed later, along with color and font.